code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "A basic energy-based laser gun that fires concentrated beams of light which pass through glass and thin metal."
 	icon_state = "laser"
 	inhand_icon_state = "laser"
-	w_class = WEIGHT_CLASS_BULKY
+	w_class = WEIGHT_CLASS_NORMAL
```

Comment:
will inherit from gun/energy, will it not?

Reply:
Yes, why was this ever there in the first place.